core/templates/dev/head/App.js

```diff
@@ -24,7 +24,15 @@ var oppia = angular.module(
     'angularAudioRecorder', 'dndLists', 'headroom', 'infinite-scroll',
     'ngAnimate', 'ngAudio', 'ngCookies', 'ngImgCrop', 'ngJoyRide', 'ngMaterial',
     'ngResource', 'ngSanitize', 'ngTouch', 'pascalprecht.translate', 'toastr',
-    'ui.bootstrap', 'ui.sortable', 'ui.tree', 'ui.validate'
+    'ui.bootstrap', 'ui.sortable', 'ui.tree', 'ui.validate',
+    'libraryPageModule', 'skillEditorModule', 'splashPageModule',
+    'showSuggestionModalForCreatorViewModule',
+    'showSuggestionModalForEditorViewModule',
+    'showSuggestionModalForLearnerViewModule',
+    'showSuggestionModalForLocalViewModule', 'stateEditorModule',
+    'storyEditorModule', 'teachPageModule', 'thanksPageModule',
+    'topicsAndSkillsDashboardModule', 'topicEditorPageModule',
+    'topicViewerPageModule',
```

Comment:
Make this in alphabetical order
 ``` showSuggestionModalForCreatorViewModule ``` should comes first then this ``` skillEditorModule ``` and some for others PTAL

Reply:
Done.